docs/source/algorithms/StartLiveData-v1.rst

```diff
@@ -20,7 +20,7 @@ simply calls :ref:`algm-LoadLiveData` at a fixed interval.
    For details on the way to specify the data processing steps, see
    :ref:`LoadLiveData <algm-LoadLiveData>`.
 
-Instructions for setting up a "fake" data stream are found `here
+Instructions for setting up a "fake" data sNtream are found `here
```

Comment:
Is `sNtream` a typo?

Reply:
It looks like a typo to me, I must have accidentally added this. Good spot.